### DISCUSSION TITLE
Flush the demo instance periodically

### DISCUSSION THREAD

**Discussion by CPUGPU** [Athou/commafeed#1014] [Ideas]
Hi, 

I use the (excellent) hosted version of Commafeed.  Occasionally I accidentally log into the demo instance, which takes several minutes to load because it's filled with junk. Closing the tab doesn't clear the credentials, so I have to log out before I can log into my account.  That can be quite a wait. Maybe you can sweep out the demo subscriptions periodically?  That will also improve the experience for new/potential user that want to try Commafeed out.  Thank you.

**Comment by Athou** [comment]
That's a good idea! I could add a scheduled task that deletes and recreates the demo account every 24h.

In the mean time you can logout by clicking on https://www.commafeed.com/logout

**Reply by CPUGPU** [reply]
Perfect.  Thank you!

**Reply by Athou** [reply]
Done in https://github.com/Athou/commafeed/commit/38a14165c3060b3b1d369f31bedad769cb4f8ee4

**Comment by CPUGPU** [comment]
After thinking about this for a while, it might be useful to include some default feeds in the refreshed demo account.  Include some in subcategories too so that feature is highlighted.   Then potential users will see something other than a blank demo, and maybe it won't fill up with junk as quickly if they do.

Also, the mark items older than x options don't work?   Update:  Didn't work on a new feed, but did after reading a couple entries.  Strange.  I will try to quantify better and open an issue if I can reliably replicate it.
